LICENSES/MIT.txt

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Didn't we want to release this under GPLv3?

Reply:
It is, kind of. We're using multiple licenses, and our code is licensed under the GNU GPLv3+.

The `LICENSES` directory contains the license texts themselves that we reference, and the MIT license is used in the `.github/workflows/reviewdog.yml` file.